tests/util/wrappers.ts

```diff
@@ -1060,14 +1065,18 @@ export class MonacoMarket {
   }
 
   async processOrderRequests(): Promise<PublicKey[]> {
-    const marketOrderRequestQueue =
-      await this.monaco.fetchMarketOrderRequestQueue(this.orderRequestQueuePk);
-
     const orderPks: PublicKey[] = [];
-    for (let i = 0; i < marketOrderRequestQueue.orderRequests.len; i++) {
+
+    // chain delays workaround
+    let tryMore = 2;
+    do {
       const orderPk = await this.processNextOrderRequest();
-      orderPks.push(orderPk);
-    }
+      if (orderPk === null) {
+        tryMore--;
+      } else {
+        orderPks.push(orderPk);
+      }
+    } while (tryMore > 0);
```

Comment:
why is this needed now? is there a delay between creating order requests and them being visible on the queue with the local validator?

Reply:
pretty much what was my experience. but even before that the loop had an issue as `processNextOrderRequest` is working with its own freshly fetched copy of the queue. so decided to get rid of the top level fetch.